### PR TITLE
docs: license

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Add license info to repo (copied from nns-js).

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2021 DFINITY LLC.
```
Comment (mstrasinskis) [inline LICENSE:189]: Why not 2022?

File: LICENSE [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   APPENDIX: How to apply the Apache License to your work.
+
+      To apply the Apache License to your work, attach the following
```
Comment (mstrasinskis) [inline LICENSE:180]: Is this a necessary part?

Comment (peterpeterparker) [reply LICENSE:180]: Sorry should have mentionned: license copied from nns-js

**Review by mstrasinskis — APPROVED** [review]
Thank you.

**Pull request merged**